images/kcp-registrar/register.sh

```diff
@@ -214,6 +214,8 @@ register() {
                 --syncer-image "ghcr.io/kcp-dev/kcp/syncer:$KCP_SYNC_TAG" \
                 --resources deployments.apps,services,ingresses.networking.k8s.io,pipelines.tekton.dev,pipelineruns.tekton.dev,tasks.tekton.dev,runs.tekton.dev,networkpolicies.networking.k8s.io \
                 --output-file "$syncer_manifest"
+            # Set a restricted security context
+            sed -i "/- name: kcp-syncer/r $(dirname "$0")/syncer-patch" "$syncer_manifest"
```

Comment:
1. `$(dirname "$0")` should be replace with `$SCRIPT_DIR` (cf other scripts)
2. I would split the bin and data in the image, instead of having everything under one folder.

Reply:
Done